src/gui/keymap.cpp

```diff
@@ -732,6 +743,29 @@ static QHash<QString, Key> getMap(KeyMap::Model model, KeyMap::Layout layout){
         map["voldn"].x -= 10;
         break;
     }
+    case KeyMap::K70_TKL:{
+        map = getMap(KeyMap::K70, layout);
+        // Very similar to the K63 but with slightly different media button positions
```

Comment:
If the only difference between the K63 and the K70TKL is just the top row, you can take the K63 map and then patch the top row on top if it. Should reduce duplicate code